controllers/horizon_controller.go

```diff
@@ -266,7 +266,7 @@ func (r *HorizonReconciler) reconcileInit(
 	//
 	// Update instance status with service endpoint url from route host information
 	//
-	instance.Status.Endpoint = apiEndpoints["public"]
+	instance.Status.Endpoint = apiEndpoints[endpoint.PublicEndpoint]
```

Comment:
`endpoint.EndpointPublic`
https://github.com/openstack-k8s-operators/lib-common/blob/main/modules/common/endpoint/endpoint.go#L42

Reply:
:facepalm: Fixed it. Also probably we can cast this to string explicitly for now until the lib-common change is approved.

Reply:
ack. Do you have the link to a lib-common PR for that change? Or just being discussed atm?